src/Domain/Bootstrap.php

```diff
@@ -3,8 +3,8 @@
 
 use Automattic\WooCommerce\Blocks\Assets\Api as AssetApi;
 use Automattic\WooCommerce\Blocks\Assets\AssetDataRegistry;
-use Automattic\WooCommerce\Blocks\Migration;
```

Comment:
Why does this (and other imports in this file) is moved? AFAIK, it doesn't relate to this PR.

Reply:
You are right, seems that after renaming the `BlockPatterns`, the IDE automatically reordered the imports alphabetically, which makes sense.
Do you think it's ok to keep those changes in this PR or it's too confusing?

Reply:
Yeah, it's ok to keep them as we don't lose any imports :).